src/route.rs

```diff
@@ -126,6 +126,55 @@ impl Route {
         self
     }
 
+    /// Set the service to be called when this route is hit.
+    ///
+    /// ```
+    /// # use actix_web::{*, dev::*, http::header};
```

Comment:
> `use actix_web::dev::*`

I am a bit concerned about the combination of the following facts:
- `Service` lies in the `actix_web::dev::*`
- `actix_web::dev` has a description that states "prelude for library developers" (i.e. it's not for *everyone*)
- `Route` is a very widely used entity.

It may lead to the fact that `actix_web::dev` will have to be used not only by the library developers but also by the end application developers, which is inconsistent.

Doesn't it make more sense now to re-export `Service` from `actix_web` top module itself and extend the description of it to explain more precisely when it should and should not be used directly (for example, with a link to this method).

Reply:
Good points, but it is already inconsistent. Anyone writing a middleware imports `dev::ServiceRequest`.

Reply:
Then I guess the description of the `dev` module should be reworked, as now it provides an expression of something you normally shouldn't really want to use 😓 

Not a subject for this PR though, I guess.

Reply:
I'm (barely) on the side of exporting `fn_service`. Also re-exporting the `always_ready!` and `forward_ready!` macros would help here and in other places.